profiler/profiler_test.go

```diff
@@ -17,6 +17,7 @@ import (
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/log"
```

Comment:
Please group these in the right place...

Reply:
Done in bc28e2a44098016836a27b3f98c52bc84b510168 - would you be able to explain the rational behind this? I'll have an easier time to remember this if I'm motivated to do it : p.